skimage/measure/_regionprops.py

```diff
@@ -530,19 +538,21 @@ def axis_minor_length(self):
     @property
     @_cached
     def moments(self):
-        M = _moments.moments(self.image.astype(np.uint8), 3)
+        M = _moments.moments(self.image.astype(np.uint8), 3, spacing=self._spacing)
         return M
 
     @property
     @_cached
     def moments_central(self):
         mu = _moments.moments_central(self.image.astype(np.uint8),
-                                      self.centroid_local, order=3)
+                                      self.centroid_local, order=3, spacing=self._spacing)
         return mu
 
     @property
     @only2d
     def moments_hu(self):
+        if not (np.array(self._spacing) == np.array([1, 1])).all():
+            raise NotImplementedError('`moments_hu` supports spacing = (1, 1) only')
```

Comment:
Ha, I wish I'd thought of this when I was trying to implement this :rofl: :+1:

Reply:
Took me a couple hours to come up with that implementation :rofl: